src/arrayMethodSort.js

```diff
@@ -4,9 +4,39 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = defaultCompare) {
+    let count;
+
+    do {
+      count = 0;
+
+      for (let i = 1; i < this.length; i++) {
+        const prev = this[i - 1];
+        const curr = this[i];
+
+        if (compareFunction(prev, curr) > 0) {
+          this[i - 1] = curr;
+          this[i] = prev;
+          count++;
+        }
+      }
+    } while (count > 0);
+
+    return this;
   };
 }
 
+const defaultCompare = (a, b) => {
+  const aStr = String(a);
+  const bStr = String(b);
+
+  if (aStr > bStr) {
+    return 1;
+  } else if (aStr === bStr) {
+    return 0;
+  }
+
+  return -1;
```

Comment:
You can compare string actually in the `if` statement.
And avoid `else if`, by adding another if and changing `return` value
```suggestion
  if (String(a) > String(b)) {
    return 1;
  }
  
  if (String(a) < String(b)) {
    return -1;
  }
  
  return 0;
```